gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua

```diff
@@ -1345,15 +1346,30 @@ function SKIN:PaintTooltipTTT2(panel, w, h)
     )
 
     if panel:HasText() then
-        drawSimpleText(
-            TryT(panel:GetText()),
-            panel:GetFont(),
-            0.5 * w,
-            0.5 * (h + sizeArrow),
-            utilGetDefaultColor(colors.background),
-            TEXT_ALIGN_CENTER,
-            TEXT_ALIGN_CENTER
-        )
+        local text = TryT(panel:GetText())
+
+        if string.find(text, "\n") then
+            -- has newlines, use drawDrawText
+            drawDrawText(
+                text,
+                panel:GetFont(),
+                10, --0.5 * w,
+                4 + sizeArrow,
+                utilGetDefaultColor(colors.background),
+                TEXT_ALIGN_LEFT
+            )
+        else
+            -- no newlines, can use drawSimpleText
+            drawSimpleText(
+                text,
+                panel:GetFont(),
+                0.5 * w,
+                0.5 * (h + sizeArrow),
+                utilGetDefaultColor(colors.background),
+                TEXT_ALIGN_CENTER,
+                TEXT_ALIGN_CENTER
+            )
+        end
```

Comment:
Hm. I see why you did it this way and this probably also works, but I'd recommend using our function `draw.GetWrappedText()` [(reference)](https://api-docs.ttt2.neoxult.de/module/draw/none/client/draw.GetWrappedText) for this.

Here's a usage sample: https://github.com/TTT-2/TTT2/blob/cb34ac03f11993486f33ccdbb007a89648814ee2/gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua#L1438-L1452

The benefit here is that it supports single and multiline, while also adding word wrap to limit the width.